src/pages/dashboard/map-container.scss

```diff
@@ -66,10 +60,13 @@ $death-color: #404856;
   border: solid lightgray 2px;
   font-weight: initial;
 }
+#toggle-container{
```

Comment:
I recommend using a class selector instead. Using IDs for CSS is considered bad practice. See https://stackoverflow.com/questions/8279132/why-shouldnt-i-use-id-selectors-in-css

We should maybe refactor all ID selectors throughout the SCSS. I've even noticed [nested IDs](https://github.com/COVID19Tracking/website/blob/dashboard-gatsby/src/pages/dashboard/map-container.scss#L101-L167) which _really_ overspecifies things. cc @goleary @beep @kevee

Reply:
Agreed!